tests/Integration/PrestaShopBundle/Controller/Admin/Sell/Order/DeliveryControllerTest.php

```diff
@@ -109,7 +112,7 @@ public function testSlipActionWithInvalidData(): void
             Response::HTTP_OK,
             $response->getStatusCode()
         );
-        $this->assertStringContainsString('This value is not valid.', $response->getContent());
+        $this->assertStringContainsString('Please enter a number.', $response->getContent());
```

Comment:
Updating symfony form changed the error message. See https://github.com/symfony/form/blob/7.0/Extension/Core/Type/NumberType.php#L63